src/shaka_scheme/system/base/String.hpp

```diff
@@ -113,53 +113,26 @@ class String {
   }
 
   friend bool operator==(const String& s1, const String& s2) {
-    if (s1.get_string().size() != s2.get_string().size()) {

```

Comment:
This code came in before we did code reviews, so this is good that it got fixed now. Good!